docs/rules/comma-style.md

```diff
@@ -8,23 +8,33 @@ In case linting is turned off, missing commas in variable declarations lead to l
 
 ## Rule Details
 
-This rule is aimed at enforcing a particular comma style in JavaScript. As such, it warns whenever it sees a variable declaration, object property and array element that does not adhere to a particular comma style. It doesn't support cases where there are line breaks before and after comma (lone commas) with in declarations, properties and elements. It also avoids single line declaration cases.
+This rule is aimed at enforcing a particular comma style in JavaScript. As such, it warns whenever it sees a variable declaration, object property and array element that does not adhere to a particular comma style.
```

Comment:
"As such, it warns whenever a variable declaration, object property, or array element does not adhere to a particular comma style." (or the respective changes if we decide to go with plural.)